pkg/app/mssql.go

```diff
@@ -0,0 +1,366 @@
+package app
+
+import (
+	"context"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+	"github.com/kanisterio/kanister/pkg/field"
+	"github.com/kanisterio/kanister/pkg/kube"
+	"github.com/kanisterio/kanister/pkg/log"
+)
+
+const (
+	mssqlWaitTimeout = 2 * time.Minute
+)
+
+type MssqlDB struct {
+	cli        kubernetes.Interface
+	namespace  string
+	name       string
+	deployment *appsv1.Deployment
+	service    *v1.Service
+	pvc        *v1.PersistentVolumeClaim
+	secret     *v1.Secret
+	dbUname    string
+	dbPass     string
+}
+
+func NewMssqlDB(name string) App {
+	return &MssqlDB{
+		name: name,
+		// These values are hard coded while creating blueprint it self
```

Comment:
I don't think password id hardcoded in blueprint.

Reply:
sorry. It's hardcoded in secret